ui/components/ui/deprecated-networks/deprecated-networks.js

```diff
@@ -1,38 +1,84 @@
-import React, { useEffect, useState } from 'react';
-import { useSelector } from 'react-redux';
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { useI18nContext } from '../../../hooks/useI18nContext';
 import {
   BackgroundColor,
   BorderRadius,
   Severity,
 } from '../../../helpers/constants/design-system';
 
-import { getCurrentChainId } from '../../../selectors';
+import { getCurrentNetwork } from '../../../selectors';
 import { getCompletedOnboarding } from '../../../ducks/metamask/metamask';
-import { BannerAlert, Box, ButtonLink } from '../../component-library';
+import { BannerAlert, Box } from '../../component-library';
 import {
+  AURORA_DISPLAY_NAME,
   CHAIN_IDS,
+  CURRENCY_SYMBOLS,
   DEPRECATED_NETWORKS,
+  NEAR_AURORA_MAINNET_IMAGE_URL,
 } from '../../../../shared/constants/network';
+import { editAndSetNetworkConfiguration } from '../../../store/actions';
+import { MetaMetricsNetworkEventSource } from '../../../../shared/constants/metametrics';
 
 export default function DeprecatedNetworks() {
-  const currentChainID = useSelector(getCurrentChainId);
-  const [isShowingWarning, setIsShowingWarning] = useState(false);
+  const { id, chainId, rpcUrl } = useSelector(getCurrentNetwork) ?? {};
+  const [isClosed, setIsClosed] = useState(false);
   const completedOnboarding = useSelector(getCompletedOnboarding);
   const t = useI18nContext();
+  const dispatch = useDispatch();
 
-  useEffect(() => {
-    if (completedOnboarding && DEPRECATED_NETWORKS.includes(currentChainID)) {
-      setIsShowingWarning(true);
-    } else {
-      setIsShowingWarning(false);
-    }
-  }, [currentChainID, completedOnboarding]);
+  if (!completedOnboarding || isClosed) {
+    return null;
+  }
 
-  const { bannerAlertDescription, actionBtnLinkURL } =
-    getDeprecationWarningCopy(t, currentChainID);
+  let props;
+  if (
+    // Goerli variants
+    chainId === CHAIN_IDS.GOERLI ||
+    chainId === CHAIN_IDS.LINEA_GOERLI ||
+    chainId === CHAIN_IDS.ARBITRUM_GOERLI ||
+    chainId === CHAIN_IDS.OPTIMISM_GOERLI
+  ) {
+    props = {
+      description: t('deprecatedGoerliNtwrkMsg'),
+      actionButtonLabel: t('learnMoreUpperCase'),
+      actionButtonProps: {
+        href: 'https://github.com/eth-clients/goerli#goerli-goerlitzer-testnet',
+        externalLink: true,
+      },
+    };
+  } else if (DEPRECATED_NETWORKS.includes(chainId)) {
+    props = { description: t('deprecatedNetwork') };
+  } else if (
+    chainId === CHAIN_IDS.AURORA &&
+    rpcUrl.startsWith('https://aurora-mainnet.infura.io/')
```

Comment:
What if a user added aurora with an RPC that is !== 'https://mainnet.aurora.dev' && !== https://aurora-mainnet.infura.io/ ?

Reply:
I don't think there's anything to warn about in that case.   Anything you're worried about?  Just targeting users who added the built-in Infura endpoint, since it would no longer work.